### DISCUSSION TITLE
Disable sign in fields to use SSO only

### DISCUSSION THREAD

**Discussion by MoltenCoffee** [directus/directus#4824] [Feature Requests]
Within my organization, we plan on only using our own OAuth/OIDC server to login. The interface still displays the "sign in" fields however. I understand it's probably a good idea to at least have some admin account to login with, but I think for UX it would be beneficial to be able to hide the login fields if one would want.

How I would see it, is that one would be able to set this with some environment variable, and that the "Login with.." buttons would take precedence in the view. Some smaller link underneath with "login with password" could still be present, which would show the normal sign in fields instead.

Would love to hear your thoughts!

**Comment by rijkvanzanten** [comment]
Makes sense to me! I don't think this is something we can default to, as it's not necessarily the expected behavior for everyone, but we could certainly add a setting to project settings that controls this 🙂

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
